developer_manual/core/ocs-share-api.rst

```diff
@@ -129,3 +130,64 @@ Statuscodes:
 * 400 - wrong or no update parameter given
 * 403 - public upload disabled by the admin
 * 404 - couldn't update share
+
+
+Federated Cloud Shares
+======================
```

Comment:
Addressed all comments. The URL is currently hardcoded, like in all other similar cases I found.